File: files/Reading.java
import java.util.Scanner;


public class Reading {
	
	private static Scanner sc;

	public static void main(String[] args) {
		sc = new Scanner(System.in);
		System.out.println("Write a line: ");
		String line = sc.nextLine();
		System.out.println("Your line:");
		System.out.println(line);
	}

}

File: files/Reading.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>
#include <string.h>

#define STR_LEN 256

char str[STR_LEN];

char* readLine() {
  fgets(str, STR_LEN, stdin);
  str[strlen(str) - 1] = '\0';
  return str;
}


void main(char** args);

void main(char** args) {
;

printf("Write a line:  \n");
char line[STR_LEN];
strcpy(line, readLine());
printf("Your line: \n");
printf("%s \n", line);
}


